config/config.yaml

```diff
@@ -25,12 +25,17 @@ RPM: 10
 
 #### for Search
 
+## Supported values: serpapi/google/serper/ddg
```

Comment:
Should Google be used as the default search?

Reply:
In the initial version, SerpAPI was used as the default search engine. Maybe using DDG might be more suitable since it's free of charge.